internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -30,6 +31,17 @@ const (
 // +kubebuilder:rbac:groups=authentication.k8s.io,resources=tokenreviews,verbs=create
 // +kubebuilder:rbac:groups=authorization.k8s.io,resources=subjectaccessreviews,verbs=create
 
+func WatchClusterTypes() []operands.WatchType {
+	return []operands.WatchType{
+		{Object: &rbac.ClusterRole{}},
+		{Object: &rbac.ClusterRoleBinding{}},
+		{Object: &core.ServiceAccount{}},
+		{Object: &core.Service{}},
+		{Object: &apps.Deployment{}},
```

Comment:
The Deployment needs to watch full resource, because it reacts to `.spec` changes:
```
{Object: &apps.Deployment{}, WatchFullObject: true},
```

Reply:
Done.